io/snmp/snmp.html

```diff
@@ -58,6 +58,60 @@
 </script>
 
 
+<script type="text/x-red" data-template-name="snmp set">
+    <div class="form-row">
+         <label for="node-input-host"><i class="fa fa-globe"></i> Host</label>
+         <input type="text" id="node-input-host" placeholder="localhost">
+    </div>
+    <div class="form-row">
+        <label for="node-input-community"><i class="fa fa-user"></i> Community</label>
+        <input type="text" id="node-input-community" placeholder="public">
+    </div>
+    <div class="form-row">
+        <label for="node-input-version"><i class="fa fa-bookmark"></i> Version</label>
+        <select type="text" id="node-input-version" style="width: 150px;">
+            <option value="1">v1</option>
+            <option value="2c">v2c</option>
+        </select>
+    </div>
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> Name</label>
+        <input type="text" id="node-input-name" placeholder="Name">
+    </div>
+</script>
+
+<script type="text/x-red" data-help-name="snmp set">
+    <p>Simple snmp Set node. Trigger by any input</p>
+    <p><code>msg.host</code> may contain the host.</p>
+    <p><code>msg.community</code> may contain the community.</p>
+    <p><code>msg.oids</code> must contain oids in JSON syntax according to net-snmp  [ { oid: "1.3.6.1.2.1.1.5.0", type: 4, value: "host1"} , { oid: ...} ]</p>
+    <p>The node will output <code>msg.payload</code> and <code>msg.oid</code>.</p>
+</script>
+
+<script type="text/javascript">
+    RED.nodes.registerType('snmp set',{
+        category: 'network-input',
+        color:"YellowGreen",
+        defaults: {
+            host: {value:""},
+            community: {value:"public",required:false},
```

Comment:
@mscbpi #324 changes the host and community to have default values, but not to require them.

```javascript
host: {value:"127.0.0.1"},
community: {value:"public"},
```
You'll need to pull from upstream again since you'll have a merge conflict.